src/sentry/api/event_search.py

```diff
@@ -117,22 +117,23 @@
 in_value         = ~r"[^(),\s]*(?:[^\],\s)]|](?=]))"
 text_value       = quoted_value / in_value
 search_value     = quoted_value / value
-numeric_value    = ~r"([-]?[0-9\.]+)([kmb])?(?=\s|\)|$|,|])"
+numeric_value    = "-"? numeric ("k"/"m"/"b")? ~r"(?=\s|\)|$|,|])"
```

Comment:
I have another PR later that actually changes the `kmb` back to a regex group to match the frontend more closely. 🤷